corehq/apps/domain/models.py

```diff
@@ -131,6 +131,7 @@ class Deployment(DocumentSchema, UpdatableSchema):
 class CallCenterProperties(DocumentSchema):
     enabled = BooleanProperty(default=False)
     use_fixtures = BooleanProperty(default=True)
+    use_user_location_as_owner = BooleanProperty(default=False)
```

Comment:
(not blocking) it could make sense to figure out a way to abstract this out somehow so that we don't end up with 3 or 4 different properties that all have undefined semantics about how they override each other or what combinations of valid states are. not totally clear to me exactly what that would look like, just a thought.